pysmt/smtlib/script.py

```diff
@@ -86,7 +86,7 @@ def serialize(self, outstream=None, printer=None, daggify=True):
 
         elif self.name in [smtcmd.CHECK_SAT, smtcmd.EXIT,
                            smtcmd.RESET_ASSERTIONS, smtcmd.GET_UNSAT_CORE,
-                           smtcmd.GET_ASSIGNMENT]:
+                           smtcmd.GET_ASSIGNMENT, smtcmd.GET_MODEL]:
```

Comment:
This was lying around, hope you don't mind me sneaking it into this PR. If you do, let me know and I will remove it.

Reply:
:+1: